api.py

```diff
@@ -17,7 +18,7 @@ def put(self):
         time.sleep(2)
         counter+=1
         
-        return 200
```

Comment:
when increment button is pressed javascript app will want to use the new value, but it will not just guess it. instead it relies on the api to return the newly computed value as the source of truth